docs/src/apps/sysvars.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Sysvar Cluster Data
+---
+
+Solana exposes a variety of cluster state data to programs via
+[`sysvar`](terminology.md#sysvar) accounts. These accounts are populated at
+known addresses published along with the account layouts in the
+[`solana-sdk`](https://docs.rs/solana-sdk/VERSION_FOR_DOCS_RS/solana_sdk/sysvar/index.html),
+and outlined below.
+
+To include sysvar data in program operations, pass the sysvar account address in
+the list of accounts in a transaction. The account can be read in your
+instruction processor like any other account. Access to sysvars is always
+*readonly*.
+
+## Clock
+
+The Clock sysvar contains data on cluster time, including the current slot,
+epoch, and estimated wall-clock timestamp. It is updated every slot.
+
+- Address: `SysvarC1ock11111111111111111111111111111111`
+- Layout: [Clock](https://docs.rs/solana-sdk/VERSION_FOR_DOCS_RS/solana_sdk/sysvar/clock/struct.Clock.html)
+
+## EpochSchedule
+
+The EpochSchedule sysvar contains epoch scheduling data that is set in genesis,
```

Comment:
Should we mention something about "epoch schedule" being aka "leader schedule"

Reply:
Do you mean clarify that epoch schedule is *not* the same as leader schedule?

Reply:
Whoops! Great typo there... yeah exactly.